json_rpc/servers/httpserver.nim

```diff
@@ -41,14 +41,19 @@ proc processClientRpc(rpcServer: RpcHttpServer): HttpProcessCallback =
 
       let body = await request.getBody()
 
+      var headers = HttpTable.init()
+      headers.add("Content-Type", "application/json; charset=utf-8")
```

Comment:
nitpick, I think you can go without the var here:

```suggestion
     let headers = HttpTable.init([("Content-Type", "application/json; charset=utf-8")])
```

Reply:
I've applied the change. Thanks @kdeme !